docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -126,6 +126,7 @@ following works:
 - github.com/opencontainers/go-digest [Apache License 2.0](https://github.com/opencontainers/go-digest/blob/master/LICENSE)
 - github.com/opencontainers/image-spec [Apache License 2.0](https://github.com/opencontainers/image-spec/blob/master/LICENSE)
 - github.com/openzipkin/zipkin-go-opentracing [MIT License](https://github.com/openzipkin/zipkin-go-opentracing/blob/master/LICENSE)
+- github.com/philhofer/fwd [MIT License](https://github.com/philhofer/fwd/blob/master/LICENSE.md)
```

Comment:
You only need to add direct dependencies (only the stuff you _added_ to go.mod).

Reply:
Ack. I way trying to fix the CI error.

Reply:
Do you know why the `make check-deps` is complaining about this dependency?

```
./scripts/check-deps.sh
--- /tmp/tmp.f2BWDwwhyY/LICENSE_OF_DEPENDENCIES.md	2021-02-07 14:23:02.275902607 +0000
+++ /tmp/tmp.f2BWDwwhyY/HEAD	2021-02-07 14:23:02.255902512 +0000
@@ -123,0 +124 @@
+github.com/philhofer/fwd
@@ -144,0 +146 @@
+github.com/tinylib/msgp


The docs/LICENSE_OF_DEPENDENCIES.md file does not contain the expected entries.
```

Reply:
I already triggered InfluxData people. Don't expect this is a problem of your code. :-) Will drop you a note as soon as it is fixed.

Reply:
Now I see. As I stated with my comment above you should not add `github.com/philhofer/fwd` but only `github.com/tinylib/msgp`. Can you please change this!?

Reply:
Ack